src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/IDOBinder.vb

```diff
@@ -1050,6 +1050,15 @@ Namespace Microsoft.VisualBasic.CompilerServices
         End Function
     End Class
 
+    Public Delegate Function SiteDelegate0(ByVal site As CallSite, ByVal instance As Object) As Object
```

Comment:
These delegates now appear to be publically exposed, whereas previously they were internal.  Is that by design?  If so, does the ref contract need to be updated, too?